src/lib/update-alternatives/model/alternative.rb

```diff
@@ -0,0 +1,67 @@
+require "cheetah"
+
+module UpdateAlternatives
+  # Represents an alternative
+  class Alternative
+    attr_reader :name
+    attr_reader :status
+    attr_reader :value
+    attr_reader :choices
+    Choice = Struct.new(:path, :priority, :slaves)
+
+    def initialize(name, status, value, choices)
+      @name = name
+      @status = status
+      @value = value
+      @choices = choices
+    end
+
+    def self.all_names
+      raw_data = Cheetah.run("update-alternatives", "--get-selections", stdout: :capture).lines
+      raw_data.map { |string| string.split.first }
+    end
+
+    def self.all
+      all_names.map { |name| load(name) }
+    end
+
+    def self.load(name)
+      raw_data = Cheetah.run("update-alternatives", "--query", name, stdout: :capture).lines
+
+      split = raw_data.find_index("\n")
+      alternative_data = raw_data[0...split]
+      choices = raw_data[split + 1..raw_data.length]
+
+      alternative = parse_alternative_data(alternative_data)
+      choices_map = parse_choices(choices)
+      new(alternative["Name:"], alternative["Status:"], alternative["Value:"], choices_map)
+    end
+
+    def self.parse_alternative_data(alternative_data)
+      alternative = {}
+      alternative_data.each do |line|
+        alternative[line.split.first] = line.split.last
```

Comment:
Why don't you strip the final colon?  Having something like `alternative["Name:"]` looks quite weird. I would expect `alternative["Name"]`.